client/src/App.js

```diff
@@ -1,29 +1,15 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-import Footer from './component/Footer.js'; 
-import 'font-awesome/css/font-awesome.min.css'
+import Footer from './component/Footer.jsx';
+import Main from './component/Main.jsx'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
```

Comment:
Since there is no real reason for this div to exist besides providing structure, you should look into Fragments in place of this

https://reactjs.org/docs/fragments.html

Reply:
I changed the `div` statements to `React.Fragment` statements, to my understanding this is to allow children and remove the vague `<div>` wrapper which is needed to render the array of elements. Can also be implemented as `<>...</>`.